src/utils/endUserMessages.js

```diff
@@ -21,6 +21,9 @@ export default {
   ERROR_SIGN_IN: 'We couldn\'t sign you in. Please try again.',
   FEEDBACK_ERROR: 'We couldn\'t process your feedback. Please try submitting again.',
   FEEDBACK_SUCCESSFUL: 'Your feedback has been successfully sent. Thank you!',
+  REFERRAL_ERROR: 'We couldn\'t process your referral. Please try submitting again',
+  REFERRAL_EMAIL_ERROR: 'The error you submitted was invalid. Pleas check again',
```

Comment:
Fix the typo

Reply:
Fixed. Will merge in after CI has done running.